hazelcast/src/test/java/com/hazelcast/internal/ascii/RestTest.java

```diff
@@ -224,7 +224,7 @@ public void addWanConfig() throws Exception {
         wanConfig.setName("test");
         WanReplicationConfigDTO dto = new WanReplicationConfigDTO(wanConfig);
         String result = communicator.addWanConfig(dto.toJson().toString());
-        assertEquals("{\"status\":\"fail\",\"message\":\"java.lang.UnsupportedOperationException: Adding new WAN config is not supported.\"}", result);
+        assertEquals("{\"status\":\"fail\",\"message\":\"Adding new WAN config is not supported.\"}", result);
```

Comment:
This was not intentional but I think it looks better now and we didn't break any behaviour as it was private API :)